app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -11,29 +11,15 @@ import androidx.lifecycle.ViewModelProvider
 import edu.uc.coffeens.stocktracker.R
 import kotlinx.android.synthetic.main.main_fragment.*
 
-class MainFragment : Fragment() {
-
-    companion object {
-        fun newInstance() = MainFragment()
-    }
+class MainFragment : Fragment(R.layout.main_fragment) {
```

Comment:
Fragment layout XML attached in the class declaration https://developer.android.com/kotlin/common-patterns

Reply:
Interesting thought, but where do you see it passed in on the constructor?  I looked through the doc provided; I don't see that reference.